src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/connection/ConnectionManager.java

```diff
@@ -59,7 +59,7 @@
 import java.util.function.Supplier;
 
 public class ConnectionManager implements AutoCloseable {
-
+    public static final String EXCLUSIVE_ROUTING_KEY = "";
```

Comment:
```suggestion
    public static final String EMPTY_ROUTING_KEY = "";
```
Maybe rename to simple

Reply:
done